packages/fiori-mcp-server/src/tools/functionalities/generate-fiori-ui-application/execute-functionality.ts

```diff
@@ -0,0 +1,88 @@
+ * @returns Application generation execution output.
+ */
+export default async function (params: ExecuteFunctionalityInput): Promise<ExecuteFunctionalityOutput> {
+    const generatorConfigValidated: GeneratorConfigOData = validateWithSchema(generatorConfigOData, params?.parameters);
+    const generatorConfig: GeneratorConfigODataWithAPI = {
+        ...PREDEFINED_GENERATOR_VALUES,
+        ...generatorConfigValidated,
+        project: {
+            ...PREDEFINED_GENERATOR_VALUES.project,
+            ...generatorConfigValidated.project
+        }
+    };
+    generatorConfig.project.sapux = generatorConfig.floorplan !== 'FF_SIMPLE';
+
+    const projectPath = generatorConfig?.project?.targetFolder ?? params.appPath;
+    if (!projectPath || typeof projectPath !== 'string') {
+        throw new Error('Please provide a valid path to the non-CAP project folder.');
+    }
+
+    const appName = (generatorConfig?.project.name as string) ?? 'default';
+    const appPath = join(projectPath, appName);
+    const targetDir = projectPath;
+    const configFileName = `${appName}-generator-config.json`;
+    const configPath = join(targetDir, configFileName);
+
+    await checkIfGeneratorInstalled();
+
+    const metadataPath = generatorConfig.service.metadataFilePath ?? join(targetDir, 'metadata.xml');
+
+    try {
+        const metadata = await FSpromises.readFile(metadataPath, { encoding: 'utf8' });
+        generatorConfig.service.edmx = metadata;
+
+        const content = JSON.stringify(generatorConfig, null, 4);
+
+        await FSpromises.mkdir(dirname(configPath), { recursive: true });
+        await FSpromises.writeFile(configPath, content, { encoding: 'utf8' });
+
+        const command = `npx -y yo@4 @sap/fiori:headless ${configFileName} --force --skipInstall`;
+        const { stdout, stderr } = await runCmd(command, { cwd: targetDir });
+        logger.info(stdout);
+        if (stderr) {
+            logger.error(stderr);
+        }
+    } catch (error) {
+        logger.error(`Error generating application: ${error}`);
+        return {
+            functionalityId: details.functionalityId,
+            status: 'Error',
+            message: 'Error generating application: ' + error.message,
+            parameters: params.parameters,
+            appPath,
+            changes: [],
+            timestamp: new Date().toISOString()
+        };
+    } finally {
+        //clean up temp config files used for the headless generator
+        if (existsSync(configPath)) {
+            await FSpromises.unlink(configPath);
+        }
+        if (existsSync(metadataPath)) {
+            await FSpromises.unlink(metadataPath);
+        }
+    }
+
+    return {
+        functionalityId: details.functionalityId,
+        status: 'Success',
+        message: `Generation completed successfully. You must run \`npm install\` in ${appPath} first, and then run the application using \`npm run start-mock\`.`,
```

Comment:
```suggestion
        message: `Generation completed successfully. You must run \`npm install\` in ${appPath} first, and then run the application using \`npm run start\`.`,
```
I think since we are generating an app from real service then preview should use the real rather than mock. Any reason not too?